src/App/templates/layout/default.html.twig

```diff
@@ -148,7 +148,7 @@
                                     </a>
                                 </p>
                                 <p class="footer-item-link margin-small">
-                                    <a href="/page/flowMiddlewares">
+                                    <a href="/page/flowMiddleware">
```

Comment:
If you modify this link (and the template), you must modify the route as well.
Now, it depends if you want to keep the existing route alive (if there might be external websites linking to it) or if it's ok to replace it.